ETAPA 3 – EXECUÇÃO DE TESTE UNITÁRIO 2 :

======================================= test session starts ========================================platform linux -- Python 3.11.11, pytest-8.3.4, pluggy-1.5.0 rootdir: /content plugins: anyio-3.7.1, langsmith-0.3.9, typeguard-4.4.2 collected 8 items test_soma_cumulativa.py FFF...F. [100%] ============================================= FAILURES =============================================_______________________________ test_soma_cumulativa_lista_positiva ________________________________def test_soma_cumulativa_lista_positiva(): > assert soma_cumulativa([1, 2, 3, 4]) == [1, 3, 6, 12]E assert [1, 3, 6, 10] == [1, 3, 6, 12] E E At index 3 diff: 10 != 12 E Use -v to get more diff test_soma_cumulativa.py:6: AssertionError _______________________________ test_soma_cumulativa_lista_negativa ________________________________def test_soma_cumulativa_lista_negativa(): > assert soma_cumulativa([-1, -2, -3, -4]) == [-1, -3, -6, -8]E assert [-1, -3, -6, -10] == [-1, -3, -6, -8] E E At index 3 diff: -10 != -8 E Use -v to get more diff test_soma_cumulativa.py:9: AssertionError _________________________________ test_soma_cumulativa_lista_mista _________________________________def test_soma_cumulativa_lista_mista(): > assert soma_cumulativa([-1, 1, -1, 1]) == [-1, 0, -1, 711]E assert [-1, 0, -1, 0] == [-1, 0, -1, 711] E E At index 3 diff: 0 != 711 E Use -v to get more diff test_soma_cumulativa.py:12: AssertionError _____________________________ test_soma_cumulativa_lista_nao_inteiros ______________________________def test_soma_cumulativa_lista_nao_inteiros(): > with pytest.raises(TypeError):E Failed: DID NOT RAISE <class 'TypeError'> test_soma_cumulativa.py:25: Failed ===================================== short test summary info ====================================== FAILED test_soma_cumulativa.py::test_soma_cumulativa_lista_positiva - assert [1, 3, 6, 10] == [1, 3, 6, 12] FAILED test_soma_cumulativa.py::test_soma_cumulativa_lista_negativa - assert [-1, -3, -6, -10] == [-1, -3, -6, -8] FAILED test_soma_cumulativa.py::test_soma_cumulativa_lista_mista - assert [-1, 0, -1, 0] == [-1, 0, -1, 711] FAILED test_soma_cumulativa.py::test_soma_cumulativa_lista_nao_inteiros- Failed: DID NOT RAISE <class 'TypeError'> ===================================4 failed, 4 passed in 0.07s ====================================